MuvrKit/MKClassifier.swift

```diff
@@ -28,6 +28,7 @@ public struct MKClassifier {
     private let neuralNet: MKForwardPropagator
     private let windowSize = 400
     private let windowStepSize = 10
+    private let scalingFactor = Float(4.0)
```

Comment:
And the minute I wrote that I realized they are actually different... I fixed it